From 5872fe5241f1327d97e7d345192870f0a5291f02 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 24 May 2020 21:16:07 +0000
Subject: [PATCH 03/12] aligned_alloc in libC is only available in glibc 2.16+

The header only implementation of aligned_alloc for fully supporting c++17 in
our gcc compilers conflicts with this.
---
 .../dist/src/libc_override_gcc_and_weak.h          |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/third_party/gperftools-2.7/dist/src/libc_override_gcc_and_weak.h b/src/third_party/gperftools-2.7/dist/src/libc_override_gcc_and_weak.h
index 6875164..9886019 100644
--- a/src/third_party/gperftools-2.7/dist/src/libc_override_gcc_and_weak.h
+++ b/src/third_party/gperftools-2.7/dist/src/libc_override_gcc_and_weak.h
@@ -214,7 +214,7 @@ extern "C" {
   void* calloc(size_t n, size_t size) __THROW     ALIAS(tc_calloc);
   void cfree(void* ptr) __THROW                   ALIAS(tc_cfree);
   void* memalign(size_t align, size_t s) __THROW  ALIAS(tc_memalign);
-  void* aligned_alloc(size_t align, size_t s) __THROW ALIAS(tc_memalign);
+//  void* aligned_alloc(size_t align, size_t s) __THROW ALIAS(tc_memalign);
   void* valloc(size_t size) __THROW               ALIAS(tc_valloc);
   void* pvalloc(size_t size) __THROW              ALIAS(tc_pvalloc);
   int posix_memalign(void** r, size_t a, size_t s) __THROW
-- 
1.7.1

